apps/web/src/styles/globals.css

```diff
@@ -54,4 +58,21 @@
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
   }
+
+  @keyframes shadow-pulse {
```

Comment:
You can define custom keyframes directly in Tailwind. See [documentation](https://tailwindcss.com/docs/animation#using-custom-values).